index.js

```diff
@@ -13,7 +13,7 @@ function Logger(settings) {
 
   const winstonSettings = _.get(settings, 'winston', {});
 
-  this._settings = _.defaultsDeep(_.cloneDeep(winstonSettings), defaultSettings);
+  this._settings = _.defaultsDeep(_.cloneDeep(winstonSettings), _.get(defaultSettings, 'winston', {}));
```

Comment:
`_.get` is not necessary since we control the defaultSettings and we know for a fact the winston prop is there.